physionet-django/project/modelcomponents/publishedproject.py

```diff
@@ -6,6 +6,7 @@
 from django.db import models
 from django.urls import reverse
 from django.utils.text import slugify
+from physionet.settings.base import StorageTypes
```

Comment:
Is this being used? I don't see it in this module. I guess it may be used in a downstream import. If so, should we add a comment to help prevent it from being removed, or instead make the import wherever it is used downstream?